### ISSUE TITLE
installing kdr on SLES 11 64bit warnings

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#61]
When installing KDR 1.0.16 on SLES 64bit we can see the following errors:

```
sles1164:/home/krmx # rpm -ihv kdr-1.0.16-centos5.11-x86_64.rpm 
Preparing...                ########################################### [100%]
   1:kdr                    ########################################### [100%]
insserv: warning: script 'kdr_agents' missing LSB tags
insserv: warning: script 'kdr_agents' missing LSB tags
insserv: Default-Start undefined, assuming default start runlevel(s) for script `kdr_agents'
insserv: warning: script 'kdr_transport' missing LSB tags
insserv: Default-Start undefined, assuming default start runlevel(s) for script `kdr_transport'
kdr_agents                0:off  1:off  2:off  3:on   4:off  5:on   6:off
insserv: warning: script 'S01kdr_agents' missing LSB tags
insserv: warning: script 'kdr_transport' missing LSB tags
insserv: warning: script 'kdr_transport' missing LSB tags
insserv: Default-Start undefined, assuming default start runlevel(s) for script `kdr_transport'
insserv: warning: script 'kdr_agents' missing LSB tags
kdr_transport             0:off  1:off  2:off  3:on   4:off  5:on   6:off
```

**Comment by sparvu** [comment]
The recorders are running just fine, problem comes from insserv script. Should be taken care in 1.0.17

**Comment by sparvu** [comment]
fixed in 1.0.17
